examples/rolling_file.rs

```diff
@@ -16,39 +16,23 @@ use logforth::append::rolling_file::NonBlockingBuilder;
 use logforth::append::rolling_file::RollingFile;
 use logforth::append::rolling_file::RollingFileWriter;
 use logforth::append::rolling_file::Rotation;
-use logforth::append::Stdout;
 use logforth::layout::JsonLayout;
 
 fn main() {
-    let rolling = RollingFileWriter::builder()
-        .rotation(Rotation::Minutely)
-        .filename_prefix("example")
-        .filename_suffix("log")
-        .max_log_files(10)
-        .max_file_size(1024 * 1024)
+    let rolling_writer = RollingFileWriter::builder()
+        .rotation(Rotation::Daily)
+        .filename_prefix("app_log")
         .build("logs")
         .unwrap();
```

Comment:
Revert these unrelated changes.

Reply:
This is intended to simplify the example. I think the original is overcomplicated and may lead the reader to confuse.

Reply:
Oops. This is in the example dir. Then it's fine.